docs/api/en/cameras/PerspectiveCamera.html

```diff
@@ -75,7 +75,26 @@ <h3>[property:Float focus]</h3>
 		</p>
 
 		<h3>[property:Float fov]</h3>
-		<p>Camera frustum vertical field of view, from bottom to top of view, in degrees. Default is *50*.</p>
+		<p>
+			Camera frustum vertical field of view, from bottom to top of view, in degrees. Default is *50*.
+			
+			The [page:.aspect aspect] direction can be vertical using the formula like this:
+			
+			<code>
+				// Calculate the vertical scale if the horizonal aspect is below the specified aspect ratio.
+				var w = 16;
+				var h = 9;
+				var aspectlimit = w / h;
+				var aspectYscale = 1;
+				if (.aspect < aspectlimit)
```

Comment:
The variable should be `aspect` (without the dot). Besides, please respect the code style of the project. Especially the spaces insides parens. For example it should look like this:

```js
if ( aspect < aspectlimit ) {

    aspectYscale = aspect / aspectlimit;

}
```